fastlane/lib/fastlane/actions/zip.rb

```diff
@@ -4,10 +4,13 @@ class ZipAction < Action
       def self.run(params)
         UI.message "Compressing #{params[:path]}..."
 
-        params[:output_path] ||= "#{params[:path]}.zip"
+        params[:output_path] ||= params[:path]
 
         absolute_output_path = File.expand_path(params[:output_path])
 
+        # Appends ".zip" if path does not end in ".zip"
+        absolute_output_path = absolute_output_path.gsub(/(?<!.zip)$/, ".zip")
```

Comment:
Why do we use a regex here, instead of `absolute_output_path.end_with?(".zip")` or even `File.extname(absolute_output_path)`? 👍